static-site/content/featured-analyses.yaml

```diff
@@ -4,16 +4,16 @@
 # - When adding or modifying descriptions, ensure it takes up at most two lines
 #   when rendered on the splash page.
 
+- name: HPAI outbreaks
+  description: "HPAI outbreak dynamics in North America: 2021-present"
+  img: hpai_in_na.png
+  url: /community/narratives/moncla-lab/nextstrain-narrative-hpai-north-america@main/HPAI-in-North-America
+
 - name: Mpox in the DRC
   description: INRB analysis of ongoing mpox clade I outbreak in the DRC
   img: mpox_drc.png
   url: /groups/inrb-mpox/clade-I
 
-- name: Mpox
-  description: Evolution and spread of mpox clades I and II
```

Comment:
_nit_

> Demote MPOX card to second row

This is only an accurate statement for a certain range of viewport widths. I'm on a larger screen which shows 6 per row, so it's still in the first row.

A more accurate description would be something like

> Demote Mpox card to below SARS-CoV-2

Reply:
Force-pushed an updated commit message.